v7/recent_posts_json/README.md

```diff
@@ -0,0 +1,22 @@
+This plug-in generates a /index.json file containing the same posts
+as /index.html. Intended to be used in JavaScripts. For example to
+promote the most recent posts at the bottom of older posts.
+
+By default, the JSON file will include:
+    {
+        <JS Date object>: {
```

Comment:
Well, … it’s JSON so storing values in JS native formats makes sense, right? Easier to work with from JavaScript. That was my thinking, at least. I usually use ISO 8601 for everything except JSON.